code/datums/wires/wires.dm

```diff
@@ -237,7 +237,7 @@ var/const/POWER = 8
 /datum/wires/proc/PulseIndex(var/index)
 	if(IsIndexCut(index))
 		return
-	playsound(holder.loc, 'sound/items/multitool_pulse.ogg', 100, 1)
+	//playsound(holder.loc, 'sound/items/multitool_pulse.ogg', 100, 1) //dampener pls
```

Comment:
Honestly if you're not gonna use this line at all you could probably just delete it instead of commenting it out.

Reply:
Added another commit.